Syntaxes/Ruby Slim.tmLanguage

```diff
@@ -576,7 +607,7 @@
 			<key>contentName</key>
 			<string>source.ruby.embedded.slim</string>
 			<key>end</key>
-			<string>(?&lt;!\\|,)$</string>
+			<string>(?&lt;!\\|,|\n)$</string>
```

Comment:
This is where my issue is coming from https://github.com/slim-template/ruby-slim.tmbundle/issues/53

If there is no `|something|` after the `do` then the syntax highlighting breaks down.